OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/FieldImmutabilityAnalysis.scala

```diff
@@ -86,7 +85,7 @@ class L0FieldImmutabilityAnalysis private[analyses] (val project: SomeProject)
             var fieldImmutabilityDependees: Set[EOptionP[Entity, Property]] = Set.empty,
             var fieldIsNotAssignable:       Option[Boolean]                 = None,
             var genericTypeParameters:      SortedSet[String]               = SortedSet.empty,
-            var innerType:                  Option[ObjectType]              = None,
+            var innerTypes:                 Set[ReferenceType]              = Set.empty,
```

Comment:
Why did this change? Can there be multiple now while there could only be one before? Why? What do multiple inner types mean compared to a single one?